challenges/profile_card_component/tailwind.config.js

```diff
@@ -5,9 +5,32 @@ module.exports = {
       colors: {
         'optimum-blue': '#009efc',
         'optimum-darkblue': '#0389e1',
+        'optimum-blue': '#009efc',
+        'optimum-darkblue': '#0389e1',
+        'dark-cyan': 'hsl(185, 75%, 39%)',
+        'dark-desaturated-blue': 'hsl(229, 23%, 23%)',
+        'dark-grayish-blue': 'hsl(227, 10%, 46%)',
+        'dark-gray': 'hsl(0, 0%, 59%)',
+      },
+      fontFamily: {
+        kumbh: ['Kumbh Sans', 'sans-serif'],
+      },
+      fontWeight: {
+        regular: 400,
+        bold: 700,
+      },
+      backgroundImage: {
+        'pattern-top-bottom':
+          "url('images/bg-pattern-top.svg'), url('images/bg-pattern-bottom.svg')",
+        'card-pattern': "url('images/bg-pattern-card.svg')",
+        'top-pattern': "url('images/bg-pattern-top.svg')",
+      },
+      backgroundPosition: {
+        'custom-top-bottom': 'right 48vw bottom 40vh, left 53vw top 40vh',
       },
     },
   },
+  //
```

Comment:
?

Reply:
I noticed a size difference for the component on mobile, but I'm having trouble detecting the difference in the desktop view, could you kindly assist?

Reply:
It's not (only) the size, but capital letter, text alignment, etc.